test/box-luatest/gh_7894_export_iproto_constants_and_features_test.lua

```diff
@@ -14,170 +14,170 @@ g.after_all(function(cg)
+                IS_ANON = 0x05,
+                IS_BOOTED = 0x06,
+                CAN_LEAD = 0x07,
+            },
 
-    -- `IPROTO_CURRENT_VERSION` constant
-    protocol_version = 4,
+            -- `iproto_type` enumeration.
+            type = {
+                OK = 0,
+                SELECT = 1,
+                INSERT = 2,
+                REPLACE = 3,
+                UPDATE = 4,
+                DELETE = 5,
+                CALL_16 = 6,
+                AUTH = 7,
+                EVAL = 8,
+                UPSERT = 9,
+                CALL = 10,
+                EXECUTE = 11,
+                NOP = 12,
+                PREPARE = 13,
+                BEGIN = 14,
+                COMMIT = 15,
+                ROLLBACK = 16,
+                RAFT = 30,
+                RAFT_PROMOTE = 31,
+                RAFT_DEMOTE = 32,
+                RAFT_CONFIRM = 40,
+                RAFT_ROLLBACK = 41,
+                PING = 64,
+                JOIN = 65,
+                SUBSCRIBE = 66,
+                VOTE_DEPRECATED = 67,
+                VOTE = 68,
+                FETCH_SNAPSHOT = 69,
+                REGISTER = 70,
+                JOIN_META = 71,
+                JOIN_SNAPSHOT = 72,
+                ID = 73,
+                WATCH = 74,
+                UNWATCH = 75,
+                EVENT = 76,
+                CHUNK = 128,
+                TYPE_ERROR = bit.lshift(1, 15),
+            },
 
-    -- `feature_id` enumeration
-    protocol_features = {
-        streams = true,
-        transactions = true,
-        error_extension = true,
-        watchers = true,
-        pagination = true,
-    },
-    feature = {
-        streams = 0,
-        transactions = 1,
-        error_extension = 2,
-        watchers = 3,
-        pagination = 4,
-    },
-}
+            -- `iproto_raft_keys` enumeration
+            raft_key = {
+                TERM = 0,
+                VOTE = 1,
+                STATE = 2,
+                VCLOCK = 3,
+                LEADER_ID = 4,
+                IS_LEADER_SEEN = 5,
+            },
 
--- Checks that IPROTO constants and features are exported correctly.
-g.test_iproto_constants_and_features_export = function(cg)
-    cg.server:exec(function(reference_table)
-        local t = require('luatest')
+            -- `IPROTO_CURRENT_VERSION` constant
+            protocol_version = 4,
 
+            -- `feature_id` enumeration
+            protocol_features = {
+                streams = true,
+                transactions = true,
+                error_extension = true,
+                watchers = true,
+                pagination = true,
+            },
+            feature = {
+                streams = 0,
+                transactions = 1,
+                error_extension = 2,
+                watchers = 3,
+                pagination = 4,
+            },
+            send = box.iproto.send,
+        }
         t.assert_equals(box.iproto, reference_table)
-    end, {reference_table})
+    end)
```

Comment:
Why?

Reply:
Beacuse at the time I was writing this test I didn't think about the fact we would need to use the `box` submodule to fill up the reference table (and in scope of this patch we already need `box.iproto.send`) — we can only do this inside the server.

Reply:
You could pass the table with constants to server and insert methods there. That would significantly reduce the diff.
But it is not really important - up to you.

Reply:
Yeah, good point, but I guess it would be better to just move this table into the `exec`, while we are here.